htdocs/comm/card.php

```diff
@@ -836,7 +836,7 @@
         $sql .= ', s.rowid as socid';
         $sql .= " FROM ".MAIN_DB_PREFIX."societe as s, ".MAIN_DB_PREFIX."expedition as e";
         $sql .= " WHERE e.fk_soc = s.rowid AND s.rowid = ".$object->id;
-        $sql .= " AND e.entity IN (".getEntity('expedition').")";
+        $sql .= " AND e.entity IN (".getEntity('commande').")";
```

Comment:
Table e is expedition, so we should still have "expedition" here?

Reply:
@eldy Good question. 
As multicompany do not manage sharable shipping settings, (as it is done on propal, order and invoice), and shipping is mandatory link with order( by definition in dolibarr we cannot create exedition without order), I wonder if we shouldn't change all getEntity('expedition') by getEntity('commande') ?
@hregis What do you think, Will you manage sharable shipping in next multicompany version ?

Reply:
We dont have to take car of what is share or not by module. We must provide information to external module of what is the type of object we want entity and we must let the module decide which value to return. So we should send 'expedition' (it is a value defined into cod comment of the getEntity method)

Reply:
@eldy Ok set toi to expedition again